serialization/php_serialized_value.ksy

```diff
@@ -0,0 +1,260 @@
+meta:
+  id: php_serialized_value
+  title: Serialized PHP value
+  application: PHP
+  file-extension: metadata.bin # Metadata stored as files in tar-based phar archives
```

Comment:
Um, do they really routinely have `metadata.bin` extension? If not, it's better not to use this field, as this will create more confusion, rather just mention that in the docs.

Reply:
Hmm. How about deprecating `file-extension` and instead introducing `filename-regexp`?

Reply:
"File extension" is a well-known thing, and many systems depend on that. Regexp might be a good idea, but I won't go as far as deprecating file extension, at least until majority of software out there will not treat it specially.

Reply:
To clarify, the files in question are called ".metadata.bin" - the name starts with a dot, as per the Unix "hidden" file convention. Depending on how you look at it, this can be considered a file with the extension "metadata.bin" (though you could argue that a leading dot doesn't count and the extension is just "bin").

It's not a general-purpose file name/extension for serialized PHP data, i. e. if you would serialize some data yourself and write it to a file, you wouldn't call it "mydata.metadata.bin".

You're probably right that having this in the `file-extension` field is more confusing than useful, I think I'll remove it.

Reply:
`file-extension` is supposed to be used by various tools like ones that analyze system file extension to action bindings. I agree that it's kind of vague, so let's leave it out for now.

I generally agree with @KOLANICH's idea on having a regexp — can I ask you to create an issue to track that?

Reply:
Done: kaitai-io/kaitai_struct#591